### PR TITLE
Add loader for container widget

### LINKED ISSUE #6257: Ability to add loading state in container component

### PR DESCRIPTION
**Pull request by stepinfwd**
closes #6257

### CODE REVIEW

File: frontend/src/Editor/Components/Container.jsx [hunk 1]
```diff
@@ -31,7 +33,7 @@ export const Container = function Container({
   return (
     <div
       data-disabled={disabledState}
-      className="jet-container"
+      className={`jet-container ${properties.loadingState && 'jet-container-loading'}`}
```
Comment (kavinvenkatachalam) [inline frontend/src/Editor/Components/Container.jsx:36]: Why do we need this additional class?

**Review by kavinvenkatachalam — CHANGES_REQUESTED** [review]
@stepinfwd Can you make the alignment center?

image

File: frontend/src/Editor/Components/Container.jsx (hunk 1 above)
Comment (stepinfwd) [reply frontend/src/Editor/Components/Container.jsx:36]: class was added to make this center aligned  , will check why its not centered

**Comment by withshubh** [comment]
docs here: https://github.com/ToolJet/ToolJet/pull/6566

**Comment by kavinvenkatachalam** [comment]
@stepinfwd Can you resolve the conflicts?

**Review by kavinvenkatachalam — DISMISSED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**